docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -325,10 +318,10 @@ Momentum Aura:
         - Control
     Description: Your strength is supernatural, bending the gravity around you.
     Type: Major
```

Comment:
To me, moving one creature as part of a 'grapple and move' maneuver feels like it could be Minor. Gathering up to a maximum of 8 enemies (if completely surrounded) and pushing them through space forcibly still feels like it should be Major.